requirements.txt

```diff
@@ -13,6 +13,7 @@ attrs==18.2.0             # via aiohttp
 babel==2.6.0              # via flower
 billiard==3.5.0.4         # via celery
 bleach==3.1.0             # via readme-renderer
+bs4==0.0.1
```

Comment:
dependencies must go in setup.py first, and these frozen requirements are generated with `pip-compile` (from the pip-tools project) -- I should mention that in the README..